legend-engine-executionPlan-execution/src/test/java/org/finos/legend/engine/plan/execution/validation/test/TestParametersValidation.java

```diff
@@ -202,7 +203,7 @@ private void testRequiredToOneParameter(String cls, List<?> validValues, List<?>
         for (Object validValue : validValues)
         {
             ExecutionState state = newExecutionState(parameter.name, validValue);
-            FunctionParametersParametersValidation.validate(Lists.immutable.with(parameter), state);
+            FunctionParametersParametersValidation.validate(Lists.immutable.with(parameter), new ArrayList<>(), state);
```

Comment:
For empty collections, we should either use eclipse facotories, or jdk Collections utility methods, and avoid specific collection implementations constructors.  

```java
Collections.emptyList();
Lists.mutable.empty();
```